bam.iobio/src/index.css

```diff
@@ -0,0 +1,11 @@
+body {
+	margin: 0;
+	font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen', 'Ubuntu', 'Cantarell', 'Fira Sans',
+		'Droid Sans', 'Helvetica Neue', sans-serif;
+	-webkit-font-smoothing: antialiased;
```

Comment:
nice!

Reply:
Useful stuff; this is just Create React App, can't take any credit for it ;)